src/Analyzer.PowerShellRuleEngine.UnitTests/PowerShellRuleEngineTests.cs

```diff
@@ -15,19 +17,22 @@ namespace Microsoft.Azure.Templates.Analyzer.RuleEngines.PowerShellEngine.UnitTe
     public class PowerShellRuleEngineTests
     {
         private readonly string templatesFolder = @"templates";
-        private static PowerShellRuleEngine powerShellRuleEngine;
+        private static PowerShellRuleEngine powerShellRuleEngineAllRules;
 
         [AssemblyInitialize]
         public static void AssemblyInitialize(TestContext context)
         {
-            powerShellRuleEngine = new PowerShellRuleEngine();
+            powerShellRuleEngineAllRules = new PowerShellRuleEngine(true);
         }
 
         [DataTestMethod]
-        [DataRow("template_and_resource_level_results.json", 12, new int[] { 1, 1, 1, 1, 8, 14, 17, 1, 17, 17, 1, 17 }, DisplayName = "Template with errors reported in both analysis stages")]
+        [DataRow("template_and_resource_level_results.json", true, 12, new int[] { 1, 1, 1, 1, 8, 14, 17, 1, 17, 17, 1, 17 }, DisplayName = "Running all the rules against a template with errors reported in both analysis stages")]
```

Comment:
What does `in both analysis stages` mean?

Reply:
oh tool errors and rule violations?

Reply:
No, PSRule detects errors when looking at the whole file (through the file path), and when looking at each resource (`pipeline.Process(resource);`)

Reply:
Oh! Maybe this is something that should be added to the ReadMe? Not sure.